<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSMainBizWindow</class>
 <widget class="QWidget" name="GBSMainBizWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1441</width>
     <height>991</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QGroupBox" name="horizontalNavi">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>72</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="imgLogo">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>27</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgBanner">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNotification">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinimize">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>10</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>10</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProfile">
    <property name="geometry">
     <rect>
      <x>1372</x>
      <y>10</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTranslate">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>实时翻译</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnData">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>数据信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProduct">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>供应链</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLive">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>远程代播</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetting">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>直播设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAI">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>大模型</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
